invenio_app_rdm/upgrade_scripts/migrate_11_0_to_12_0.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-App-RDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Record migration script from InvenioRDM 10.0 to 11.0.
+
+Disclaimer: This script is intended to be executed *only once*, namely when
+upgrading from InvenioRDM 10.0 to 11.0!
+If this script is executed at any other time, probably the best case scenario
+is that nothing happens!
+"""
+
+from click import secho
+from invenio_db import db
+from invenio_communities.communities.records.api import Community
+
+
+def execute_upgrade():
+    """Execute the upgrade from InvenioRDM 11.0 to 12.0.beta
+
+    Please read the disclaimer on this module before thinking about executing
+    this function!
+    """
+
+    def migrate_review_policy(community_record):
+        review_policy = community_record["access"].get("review_policy", "closed")
+        community_record["access"]["review_policy"] = review_policy
+
+    secho("Starting data migration...", fg="green")
+    communities = Community.model_cls.query.all()
+
+    for community_metadata in communities:
```

Comment:
Maybe I am bit mistaken, but why this recreating the community with the `.data`?

Isn't it enough to do:
```
for community in CommunityMetadata.query.all():
    migrate_review_policy(community)
    community.commit()
```

Reply:
I think it is not enough because communities return the db rows, not the metadata api class, so without that I cannot do community.commit(). I think simply the inner for loop object name was misleading, I changed it